pytest.ini

```diff
@@ -1,3 +1,3 @@
 [pytest]
-addopts = --cov=podcastparser --cov-report html --doctest-modules
+adopts = --cov=podcastparser --cov-report html --doctest-modules
```

Comment:
I'm pretty sure this is supposed to be `addopts` (ad in add options?). Please change back?

Reply:
I changed this as atleast when I was testing the python tests module didnt work unless I changed it to this